redaxo/src/addons/structure/lib/buttons/article2category/button_article2category.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Generate button to change article type to category
+ *
+ * @package redaxo\structure
+ */
+class rex_button_article2category extends rex_structure_button
```

Comment:
@gharlan da diese klassen hier eigentlich nur view und viewlogik enthalten könnte man draus vllt auch eine subclass von rex_fragment machen.. das html dazu in einem fragmentfile und die logik im der subclass ... was meinst du?